code-nation/raise_nation
# Raisely nation workflow

**IMPORTANT NOTE: `6-donation-data` branch must be merged to this branch first before it's ready for merging.**

To Test:

1. Go to https://raise-nation-staging.herokuapp.com/ with the following credentials:

```
Login: test@test.com1
Password: <same as login>
```

2. Navigate by clicking `Workflows` from the sidebar.

3. You can see listing of Workflows if it's available.
4. Click on `New Workflow` button.
5. You can select from `Workflow data type` if it's Nation to Raisely or vice versa.
6. Fill-up the form and select the source and target.
7. Populate donor tags and recurring donor tags by hitting Enter or Space for each tag.
8. Set active or inactive by toggling the checkbox
9. Click on Save button to save the workflow.
10. If the source record is correctly configured, it will automatically populate webhook instance from the source's API. You can  check the webhook instance from Nation admin dashboard or Raisely Campaign settings -> API and webhooks page to confirm.
